### PR TITLE
fix(stream): change XREADGROUP response for empty read response

### PR DESCRIPTION
**Pull request by jonathanc-n**
references #2568 

Changed XReadGroup response for empty responses to match redis behaviour.

### CODE REVIEW

**Comment by torwig** [comment]
@jonathanc-n Should we change existing tests or add extra tests for this case if they don't exist?

**Comment by jonathanc-n** [comment]
@torwig changes in this particular pr are negligible so probably doesn't need a test, but for the other half of the issue I believe there should be a test.

**Comment by torwig** [comment]
@jonathanc-n How to be sure that bug was fixed? Exactly, write a test :)
 
There is a classic flow for fixing a bug:
1. Write a test that fails.
2. Fix a bug.
3. Assert that the test passes -> it will prevent any regression.

**Comment by jonathanc-n** [comment]
I see, thank you for letting me know! :)

**Comment by jonathanc-n** [comment]
@torwig The test is added

**Review by torwig — DISMISSED** [review]

**Comment by jonathanc-n** [comment]
@torwig I made some changes to the stream, hopefully it passes all the checks now

File: src/commands/cmd_stream.cc [hunk 1]
```diff
@@ -31,6 +31,7 @@
 #include "status.h"
 #include "time_util.h"
 #include "types/redis_stream.h"
+#include "types/redis_string.h"
```
Comment (PragmaTwice) [inline src/commands/cmd_stream.cc]: seems we actually don't need it.

Comment (jonathanc-n) [reply src/commands/cmd_stream.cc]: sorry bout that, thought i would need it for the bulk string

**Review by torwig — APPROVED** [review]

**Review by PragmaTwice — APPROVED** [review]

**Comment by jonathanc-n** [comment]
@PragmaTwice Do you know why this might be failing? I am not too sure myself

**Comment by PragmaTwice** [comment]
It keeps failing even after multiple retries.

I'll look into it soon.

**Pull request merged**